xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/displayers/DisplayerHtml.vue

```diff
@@ -74,4 +74,12 @@ export default {
 
 <style>
 
+/*
+ * Fix styles of xwiki html widgets when needed
+ * So that it renders best as displayer
+ */
+.xwiki-livedata .displayer-html .breadcrumb {
+  margin-bottom: 0;
```

Comment:
You should go all the way and remove also the padding and the background. See https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/table/livetable.css#L396 .